build.rs

```diff
@@ -67,6 +68,12 @@ fn main() {
         Some(_) | None => (),
     }
 
+    // Some ABIs need to redirect time related symbols to their time64
+    // equivalents. See #2088 and #1848 for more information.
+    if is_musl_time64_abi() {
+        set_cfg("musl_time64_abi");
```

Comment:
We should set two separate configs here:

- `musl_redir_time64` (`musl_time64_abi`) which corresponds to musl's _REDIR_TIME64
- `linux_time_bits64` which corresponds to `__USE_TIME_BITS64` in uapi

We need this to be split because musl and glibc share the kernel uapi